src/components/CreateResetPassword/CreateResetPasswordControl.js

```diff
@@ -66,6 +70,12 @@ class CreateResetPasswordControl extends Component {
       }
     } = this.props;
 
+    const { isSuccessfulPasswordChange } = this.state;
+
+    if (isSuccessfulPasswordChange) {
+      return <SuccessPage />;
```

Comment:
In this case you don't change route. So it is not a page. It should be renamed here and in correspond component also